gh-weblog/components/Admin.jsx

```diff
@@ -42,7 +42,7 @@ export default React.createClass({
                   type="text"
                   placeholder="yourname"
                   value={this.state.user}
-                  onChange={(evt) => this.update(`user`, evt)}
```

Comment:
this is one thing I liked about React:  typing a letter is a change, whether you hit enter or not, so the distinction between the `input` and `change` events was one of the rare few places where not following the browser made sense for what React was (as it doesn't follow the browser's "events as/after the UI changes" model, but uses its own "events _before_ the UI changes" model)

Reply:
I like the explicitness and ability to choose which one I actually want, although I suppose a really strict two-way data binding approach works best when everything is fully synchronized.

Reply:
You only need both if you want to work "in the browser model and the react model at the same time", which IMO is a bit like using jquery and react at the same time: pick one or the other, not both.